cli/src/cluster_query.rs

```diff
@@ -97,6 +113,14 @@ impl ClusterQuerySubCommands for App<'_, '_> {
     }
 }
 
+pub fn parse_catchup(matches: &ArgMatches<'_>) -> Result<CliCommandInfo, CliError> {
+    let node_pubkey = pubkey_of(matches, "node_pubkey").unwrap();
```

Comment:
Seems like it would be helpful to use the cli config `--keypair` by default if a pubkey is not provided... Something like this: https://github.com/solana-labs/solana/blob/a3a830e1abac1116c387b3f021fa81e3eb82f102/cli/src/cli.rs#L550
(In which case the clap arg `node_pubkey` should remain optional)

Reply:
I'm going to keep the explicit `node_pubkey` and make it required.  I feel that the implicit use of `--keypair` could result in confusion.   But after we test drive this command a little if it becomes awkward to not have `--keypair` I'm happy to add in that support